### DISCUSSION TITLE
compilation issue

### DISCUSSION THREAD

**Discussion by Xiaoguangissleeping** [CompuCell3D/CompuCell3D#773] [Q&A]
Hello,

I followed the online manual to do compilation https://compucell3ddevelopersmanual.readthedocs.io/en/master/building_core_cc3d_cpp_code_linux.html. 

but after running this command: `cmake -S /home/m/src-cc3d/CompuCell3D/CompuCell3D -B /home/m/src-cc3d/CompuCell3D_build -DPython3_EXECUTABLE=/home/m/miniconda3/envs/cc3d_compile/bin/python -DNO_OPENCL=ON  -DBUILD_STANDALONE=OFF -DOPENGL_gl_LIBRARY=/usr/lib/x86_64-linux-gnu/libGL.so -DOPENGL_glx_LIBRARY=/usr/lib/x86_64-linux-gnu/libGLX.so -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=/home/m/src-cc3d/CompuCell3D_install` , I got a warning about PDEsolvers.
![PDEsolvers_warning](https://github.com/user-attachments/assets/fde72854-31fe-448b-a34d-e9ce78bdd4a9)

Then I continue to run the install command: 
`cd /home/m/src-cc3d/CompuCell3D_build`
`make -j 8`
The compilation failed at 97%. 
![failed](https://github.com/user-attachments/assets/5e7b75d5-ce37-45e4-ba1c-8437787224a4)
Could anyone help me with this? Many thanks!

Regards,
Elle

**Comment by maciekswat** [comment]
Your command needs adjustments because the paths beginning with  `/home/m/` likely do not exist on your system. Check what username you are using on your system, and I assume that this is the user `xyz`, replace all paths starting with `/home/m` with `/home/xyz`, so 
cmake -S /home/xyz/src-cc3d/CompuCell3D/CompuCell3D -B /home/m/src-cc3d/CompuCell3D_build -DPython3_EXECUTABLE=/home/xyz/miniconda3/envs/cc3d_compile/bin/python ...
